static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -115,13 +115,15 @@ interface ConsoleMessageProps extends MessageFormatterProps {
   hasOccurred: boolean;
   isActive: boolean;
   isLast: boolean;
+  isSelected: boolean;
```

Comment:
What's the difference between `isSelected` and `isActive`?

Reply:
`isActive` is being calculated based on the hover time, while `isSelected` is using the current replay time

Reply:
It looks like isActive is the hover state over an item vs whatever item is 'current' based on the timestamp or bread crumb. I would rename this to isCurrent for clarity since selected is a different type of state/action. current also implies that out of a list there is one item that can be considered current at a time